tests/src/test/scala/parser/LinePositionSuite.scala

```diff
@@ -0,0 +1,68 @@
+import org.scalatest._
+import java.net._
+import java.io._
+
+import scala.meta._
+import scala.meta.dialects.Scala211
+import scala.meta.internal.{ast => impl}
+import scala.meta.tql._
+
+class LinePositionSuite extends ParseSuite {
+  var dir = new File(new File(System.getProperty("sbt.paths.tests.source")).getAbsolutePath)
+  def isProjectRoot(dir: File) = dir != null && new File(dir.getAbsolutePath + File.separatorChar + "project" + File.separatorChar + "build.scala").exists
+  while (dir != null && !isProjectRoot(dir)) dir = dir.getParentFile
+  test("ProjectDir (" + dir.getAbsolutePath + ")")(assert(isProjectRoot(dir)))
+
+  val ignoredFiles = List(
+    "build.scala",
+    "Token.scala",
+    "Api.scala"
+  )
```

Comment:
Build.scala issue some parse exception, probably due to its special syntax:

>[info]   scala.meta.ParseException: XML literals are not supported at < (1751..1751)
>[info]   at scala.meta.internal.parsers.Reporter$class.syntaxError(Reporter.scala:13)
>[info]   at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:18)
>[info]   at scala.meta.internal.parsers.Parser.xmlLiteral(Parsers.scala:1230)
>[info]   at scala.meta.internal.parsers.Parser$$anonfun$simpleExpr$1.apply(Parsers.scala:1254)
>[info]   at scala.meta.internal.parsers.Parser$$anonfun$simpleExpr$1.apply(Parsers.scala:1245)
>[info]   at scala.meta.internal.parsers.Parser.atPos(Parsers.scala:277)
>[info]   at scala.meta.internal.parsers.Parser.autoPos(Parsers.scala:294)
>[info]   at scala.meta.internal.parsers.Parser.simpleExpr(Parsers.scala:1245)
>[info]   at scala.meta.internal.parsers.Parser.prefixExpr(Parsers.scala:1221)
>[info]   at scala.meta.internal.parsers.Parser.postfixExpr(Parsers.scala:1207)


Token.scala fails, because apparently the name of

```scala
@token class `macro`(start: Int) extends Keywords
```

is named "macro" without `.

/semantic/Api.scala contains something of the form:

```scala
internalFilter[T](that => {
              def thisDenot = thisName.denot.require[h.Denotation.Precomputed]
              def thatDenot = that.require[impl.Member].name.require[impl.Name].denot.require[h.Denotation.Precomputed]
              scala.util.Try(thisDenot.symbol == thatDenot.symbol).getOrElse(false)
            }) match {
		...
	    }
```

The starting line of the match contains `internalFilter[T](that => {` only, and therefore does not pass the test for `match`. To keep the test simple, I decided to avoid checking the Api.scala files.